docs/rules/no-missing-link-fragments.md

```diff
@@ -1,68 +1,83 @@
 # no-missing-link-fragments
 
-Disallow link fragments that don't exist in the document.
+Ensures that link fragments (URLs that start with `#`) reference valid headings or anchors in the document. This rule helps prevent broken internal links.
 
 ## Rule Details
```

Comment:
For our documentation format, we also have a "Background" section that gives some context as to what the rule is meant to address. Please see other rules in the plugin for examples.